bindings/swig/module.i

```diff
@@ -6,49 +6,49 @@
 
 %include <exception.i>
 %include <std_map.i>
-%include <std_pair.i>
-%include <std_set.i>
-%include <std_shared_ptr.i>
-%include <std_string.i>
 %include <std_vector.i>
+%include <std_string.i>
 
+%import(module="libdnf.common_types") "common_types.i"
 
-%exception {
+%exception %{
```

Comment:
Please why % is here?

Reply:
It is equivalent for SWIG interpret in this context.
But for me it is cleaner. It distinguishes between SWIG block and C++ code block.